packages/voice/src/audio/TransformerGraph.ts

```diff
@@ -101,71 +101,19 @@ export class Node {
 }
 
 // Create a node for each stream type
-const NODES = new Map<StreamType, Node>();
-for (const streamType of Object.values(StreamType)) {
-	NODES.set(streamType, new Node(streamType));
-}
+let NODES: Map<StreamType, Node> | null = null;
 
 /**
  * Gets a node from its stream type.
  *
  * @param type - The stream type of the target node
  */
 export function getNode(type: StreamType) {
-	const node = NODES.get(type);
+	const node = (NODES ??= initializeNodes()).get(type);
```

Comment:
This looks like it will no longer work, as it will just constantly recurse 👀

Reply:
Not if it uses the local map instead of calling this helper, will also improve efficiency a little. https://github.com/discordjs/discord.js/pull/9918#discussion_r1382622692

Reply:
Fixed now!